cross/sqlite/Makefile

```diff
@@ -15,8 +15,15 @@ GNU_CONFIGURE = 1
 CONFIGURE_ARGS  = --disable-static
 CONFIGURE_ARGS += --disable-static-shell
 
+PRE_CONFIGURE_TARGET = sqlite-pre-configure
+
 ADDITIONAL_CFLAGS  = -O
 ADDITIONAL_CFLAGS += -DSQLITE_ENABLE_COLUMN_METADATA
 ADDITIONAL_CFLAGS += -DSQLITE_ENABLE_UNLOCK_NOTIFY
 
 include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: sqlite-pre-configure
+
+sqlite-pre-configure:
+	$(RUN) autoreconf -f -i
```

Comment:
please revert the changes in this file.

sqlite was updated on the master to version 3.39.3 (3390300, 2022)
your PR would downgrade it.

BTW: If you use the SynoCommunity docker image as dev. environment (or set it up regarding the README.md) there is no need to call autoreconf to build this.

Reply:
I've reverted both commits.